<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModInfoForm</class>
 <widget class="QWidget" name="ModInfoForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="modImageLabel">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Resources/mod.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="modNameLabel">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mod Name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="modSizeLabel">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mod size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="modSizeValueLabel">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mb</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="numberOfFilesLabel">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Number of files:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numberOfFilesValueLabel">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="addTimeLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add time:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="addTimeValueLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="dateAddedLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Date Added:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="dateAddedValueLabel">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListWidget" name="modFilesListWidget">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="OkPushButton">
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
